docs/user/resources/05-metricpipeline.md

```diff
@@ -96,10 +96,18 @@ For details, see the [MetricPipeline specification file](https://github.com/kyma
 | **input.&#x200b;runtime.&#x200b;resources**  | object | Describes the Kubernetes resources for which runtime metrics are scraped. |
 | **input.&#x200b;runtime.&#x200b;resources.&#x200b;container**  | object | Configures container runtime metrics scraping. |
 | **input.&#x200b;runtime.&#x200b;resources.&#x200b;container.&#x200b;enabled**  | boolean | If enabled, the runtime metrics for the resource are scraped. The default is `true`. |
+| **input.&#x200b;runtime.&#x200b;resources.&#x200b;daemonset**  | object | Configures DaemonSet runtime metrics scraping. |
+| **input.&#x200b;runtime.&#x200b;resources.&#x200b;daemonset.&#x200b;enabled**  | boolean | If enabled, the runtime metrics for the resource are scraped. The default is `false`. |
```

Comment:
The text is autogenerate from [here](https://github.com/kyma-project/telemetry-manager/blob/main/apis/telemetry/v1alpha1/metricpipeline_types.go#L140). Since its part of a function which is used at multiple places we cant change it per resource.